src/App.tsx

```diff
@@ -8,55 +8,163 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { customFetch } from './utils/fetchClient';
+import { User } from './types/User';
+import { ErrorType } from './types/ErrorType';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [isErrorType, setErrorType] = useState<ErrorType | null>(null);
```

Comment:
The variable isErrorType suggests a boolean type, but it's assigned a value of ErrorType or null. Consider renaming it to something more appropriate, like errorType.